layouts/partials/hooks/body-end.html

```diff
@@ -2,3 +2,5 @@
   {{ partial "script.html" (dict "src" "js/tracing.js") -}}
 {{ end -}}
 {{ partial "script.html" (dict "src" "js/navScroll.js") -}}
+{{ $js := resources.Get "js/scroll-to-top.js" | minify | fingerprint }}
+<script src="{{ $js.Permalink }}"></script>
```

Comment:
```suggestion
{{ partial "script.html" (dict "src" "js/scrollToTop.js") -}}
```

Make use of what is there already, also be consistent with the file naming

Reply:
Indeed, done!